app/controllers/peeps.rb

```diff
@@ -0,0 +1,9 @@
+get '/peeps/new' do
+  erb :"peeps/new"
+end
+
+post '/peeps' do
+  content = params["content"]
```

Comment:
Also try to be consistent with how you access your hashes - in other controllers you're accessing with symbols, and here with a string. Both are fine, but when both being used it makes me wonder if there's something weird going on with params, which there isn't